rs-commerce/src/components/login-page-form/login-page-form.tsx

```diff
@@ -0,0 +1,187 @@
+      } else {
+        setPasswordTips('');
+        setIsPasswordValid(true);
+      }
+    }
+  };
+
+  const checkCanSubmit = (): void => {
+    if (isEmailValid && isPasswordValid) {
+      setIsCanSubmit(true);
+    } else {
+      setIsCanSubmit(false);
+    }
+  };
+
+  useEffect(() => {
+    checkIsEmailValid();
+    checkIsPasswordValid();
+  }, [emailValue, passwordValue]);
+
+  useEffect(() => {
+    checkCanSubmit();
+  }, [isEmailValid, isPasswordValid]);
+
+  const handleInputEmail = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setEmailTouched(true);
+    setEmailValue(e.target.value);
+  };
+
+  const handleInputPassword = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setPasswordTouched(true);
+    setPasswordValue(e.target.value);
+  };
+
+  const openEye = (): void => {
+    setIsOpenEye(!isOpenEye);
+  };
+
+  const handleBtnLogin = (): void => {
+    if (isEmailValid && isPasswordValid) {
+      console.log('handle login with data: ', { email: emailValue, password: passwordValue });
+    }
+  };
+
+  const loginPageBtnStyle = (): string =>
+    isCanSubmit
+      ? `${style['login-page-btn']}`
+      : `${style['login-page-btn']} ${style['btn-not-active']}`;
+
+  const eyeStyle = (): string =>
+    isOpenEye ? `${style['password__eye']} ${style['eye-open']}` : `${style['password__eye']}`;
+
+  const passwordType = (): string => (isOpenEye ? 'text' : 'password');
+
+  return (
+    <div className={style['login-page-form']}>
+      <div className={style['login-page-form-title']}>{LOGIN_PAGE_TEXT.titleForm}</div>
+      <div className={style['login-page-form-register']}>
+        {LOGIN_PAGE_TEXT.linkRegistration}
+        <Link to="/registration" className={style['text-blue']}>
+          {LOGIN_PAGE_TEXT.linkTo}
+        </Link>
+      </div>
+      <div className={style['input-wrapper']}>
+        <div className={style['input-field-name']}>{LOGIN_PAGE_TEXT.titleEmail}</div>
+        <input
+          className={style['input-field']}
+          id="login-email"
+          placeholder={LOGIN_PAGE_TEXT.placeholderEmail}
+          value={emailValue}
+          onChange={handleInputEmail}
+        />
+        <div className={style['input-field-tips']}>{emailTips}</div>
+      </div>
+      <div className={style['input-wrapper']}>
+        <div className={eyeStyle()} onClick={openEye} role="presentation" />
+        <div className={style['input-field-name']}>{LOGIN_PAGE_TEXT.titlePassword}</div>
+        <input
+          className={style['input-field']}
+          id="login-password"
+          type={passwordType()}
+          placeholder={LOGIN_PAGE_TEXT.placeholderPassword}
+          value={passwordValue}
+          onChange={handleInputPassword}
+        />
+        <div className={style['input-field-tips']}>{passwordTips}</div>
+      </div>
+      <div className={loginPageBtnStyle()} onClick={handleBtnLogin} role="presentation">
+        {LOGIN_PAGE_TEXT.loginPageBtn}
+      </div>
```

Comment:
Better use `button`  tag for buttons. Also I propose to use separate component for button also to reuse it because some buttons in project have the same style